cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -214,15 +226,49 @@ install_cross_toolchain_pkg() {
   info "Configuring environment variables for cross-compilation"
   export PATH="/build/bin:${PATH}"
   export SYSROOT="/build/usr/x86_64-cros-linux-gnu"
-  export CC="x86_64-cros-linux-gnu-gcc"
+}
+
+set_compilation_env() {
+  # Get toolchain_info path from COS GCS bucket
+  local -r tc_info_file_path="${COS_DOWNLOAD_GCS}/${BUILD_ID}/${TOOLCHAIN_INFO_FILENAME}"
+  info "Obtaining toolchain_info file from ${tc_info_file_path}"
+  local -r download_tc_info_url="$(curl -sfS "${tc_info_file_path}")"
+
+  # try to download toolchain_info if present
```

Comment:
Line 238-247 can be merged with line 214-223 by creating a helper function to download blob from url.